prime-angular-frontend/src/app/config/config.service.ts

```diff
@@ -0,0 +1,168 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { Config } from './config.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ConfigService {
+  private configuration: Config;
+
+  constructor() { }
+
+  public get advancedPractices() {
+    return [...this.configuration.advancedPractices];
+  }
+
+  public get colleges() {
+    return [...this.configuration.colleges];
+  }
+
+  public get countries() {
+    return [...this.configuration.countries];
+  }
+
+  public get jobNames() {
+    return [...this.configuration.jobNames];
+  }
+
+  public get licenses() {
+    return [...this.configuration.licenses];
+  }
+
+  public get organizationNames() {
+    return [...this.configuration.organizationNames];
+  }
+
+  public get provinces() {
+    return [...this.configuration.provinces];
+  }
+
+  /**
+   * Load runtime configuration.
+   *
+   * @returns {Promise<Config>}
+   * @memberof ConfigService
+   */
+  public async load(): Promise<Config> {
```

Comment:
Can't wait to hook this up to services so we don't have these hardcoded values here.